status-im/status-react
# community link unfurling

fixes #11775

### Summary

Unfurl community links
![Screenshot 2021-04-16 at 14 28 15](https://user-images.githubusercontent.com/5786310/115018152-32e96c80-9ec0-11eb-9d7c-8386cecaad2e.png)

Page for community that we don't know:
![Screenshot 2021-04-16 at 14 13 40](https://user-images.githubusercontent.com/5786310/115018205-4268b580-9ec0-11eb-914a-139ffc1ed7fd.png)



### Review notes
Added previews for universal community links `https://join.status.im/c/COMMUNITY_ID`.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
chats, links unfurling

##### Functional

- 1-1 chats
- public chats
- group chats


### Steps to test

- Open Status
- Join `community-unfurling-test4` public channel
- Make sure you see `Status` and `Watercooler` links unfurled (after few seconds, when community info request succeeded)


status: ready
